toolkit/themes/shared/design-system/config.js

```diff
@@ -74,7 +74,7 @@ const createDesktopFormat = surface => args => {
       surface,
       args,
     })
-  );
+  ).replaceAll("-base", "");
```

Comment:
This is just a placeholder the real solution is in https://github.com/FirefoxUX/gecko-dev/pull/8